panel/src/components/View/Menu.vue

```diff
@@ -40,42 +40,57 @@
 					theme="love"
 					variant="filled"
 				/>
-				<k-activation :status="$panel.license" />
+				<k-activation :status="license" />
 			</menu>
 		</div>
 
 		<!-- Collapse/expand toggle -->
 		<k-button
-			:icon="$panel.menu.isOpen ? 'angle-left' : 'angle-right'"
-			:title="$panel.menu.isOpen ? $t('collapse') : $t('expand')"
+			:icon="isOpen ? 'angle-left' : 'angle-right'"
+			:title="isOpen ? $t('collapse') : $t('expand')"
 			size="xs"
 			class="k-panel-menu-toggle"
-			@click="$panel.menu.toggle()"
+			@click="$emit('toggle')"
 		/>
 	</nav>
 </template>
 
 <script>
 /**
- * @since 4.0.0
+ * @displayName PanelMenu
+ * @since 6.0.0
  * @unstable
  */
 export default {
+	props: {
+		isHovered: Boolean,
+		isOpen: Boolean,
+		items: {
+			type: Array,
+			default: () => []
+		},
+		license: String,
+		searches: {
+			type: Object,
+			default: () => ({})
+		}
```

Comment:
I wonder if this should be even more "stupid" and just be a boolean "search" prop. We don't do anything else with the searches object and only check its length. That could also be done outside of the menu.